random-referral/Gemfile

```diff
@@ -38,8 +38,9 @@ gem "bootsnap", require: false
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
-  gem 'rspec-rails', '~> 6.0.0.rc1'
+  gem "debug", platforms: %i[mri mingw x64_mingw]
+  gem "rspec-rails", "~> 6.0.0.rc1"
+  gem "standard"
```

Comment:
Can you add a note to the README about running this locally

Reply:
Ah yes, I forgot to update the README, thanks for the reminder!